Source/Scene/PostProcessStageCollection.js

```diff
@@ -294,6 +294,9 @@ define([
                 var stages = arraySlice(this._stages);
                 while (stages.length > 0) {
                     var stage = stages.pop();
+                    if (!defined(stage)) {
```

Comment:
Is it expected that a stage in this list would be undefined? Just want to make sure this isn't a deeper issue.

Reply:
Yes. See this line in `remove`: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/PostProcessStageCollection.js#L402

They are set to `undefined` and removed on the next update.

Reply:
Ah, ok.